engine/classes/ElggFile.php

```diff
@@ -80,6 +80,18 @@ public function getFilenameOnFilestore() {
 	}
 
 	/**
+	 * Returns path relative to the filestore (data directory) root
+	 *
```

Comment:
Is there any way to implement a custom filestore at this point? Will it ever be an option given a few thousand plugins are built around filestore being the data root?